### DISCUSSION TITLE
Preview of each color list

### DISCUSSION THREAD

**Discussion by bytrangle** [meodai/color-names#151] [Ideas]
Hi. What do you think about displaying a preview of what are the most common colors named in each color list.

Examples of common colors:
- warm red (blood color)
- cool red (crimson or magenta)
- cool blue (sea blue)
- warm blue (denim blue)
- cool yellow (lemon)
- warm yellow (orange)

There are many color lists in Color Names and inspecting them one by one takes time. 

In addition, the way colors are named in each list seems to reflect a different vibe, e.g. Best Of is very edgy, Universal Color Language is descriptive and plain, and The Color Thesaurus is poetic etc. Meanwhile, users may not know the names for their colors, but they often have a rough idea of what the names should convey.

I think displaying a preview will significantly save users' times.

**Comment by meodai** [comment]
@bytrangle thats a neat idea. But since I have no usage stats, I don't know what names are "popular" I think having a demo-page would solve that: https://codepen.io/meodai/full/vYddvLB

**Reply by bytrangle** [reply]
That's a good demo. [Here](https://codepen.io/trangthule/full/jOpOrdv), I've added more colors to make it clear what the personality of each list is.

Having stats for the most popular colors is helpful, but not necessary IMO. I'd go with three primary colors (red, blue, yellow) and two secondary but super common (green, orange). Can add grey, or warm and cool versions of each color if required.

![color-names-preview](https://user-images.githubusercontent.com/28924121/208711323-c95abd9e-05e5-4e7a-aadb-b1b61278a5fc.gif)

**Comment by meodai** [comment]
@bytrangle thats a very helpful demo. If you want to add it to the readme, I will gladly accept the pull-request
